website/docs/support/known-issues.md

```diff
@@ -25,3 +25,5 @@ Below is a list of known Corso issues and limitations:
   not be inheritable by children.
 
 * Link shares with password protection can't be restored.
+
+* All replies in a Teams Conversation are removed from backup when the root message gets deleted.
```

Comment:
Not sure if this explanation is sufficient.  Thoughts?  @gmatev, pinging for scrutiny here.

Reply:
@ryanfkeepers - can you clarify (here in the PR comment ) how the user gets into the situation and what the impact is? Based on that - can review what should be added to known-issues.

Reply:
Sure.  Easiest way to explain is with this image.
<img width="335" alt="image" src="https://github.com/alcionai/corso/assets/104464746/8d36c401-8496-4372-9976-434dc21cebf1">

What's going on here is that I created a conversation, replied to it multiple times, and then deleted the initial message.  In the Teams app, the conversation survives with a "deleted" root message and all replies intact.

However, when utilizing the API, these properties are unavailable.  Replies are only retrievable with a message ID, and the deleted message ID now returns 404 for both GetMessage and GetMessageReplies endpoints.

This could lead users to expect that all replies are backed up and restorable, while in Corso I'm unable to get any reply in the conversation.

Reply:
@ryanfkeepers as phrased, I think this is pretty confusing as stated. 

 I would probably remove this all together or replace it with something along the lines of "Message replies are only backed up when their corresponding parent message is still available at the time of backup".

Reply:
will do, thanks.